sdcm/utils/adaptive_timeouts/load_info_store.py

```diff
@@ -57,19 +57,18 @@ def __init__(self, remoter: RemoteCmdRunner, name: str, scylla_version: str):
         self.remoter = remoter
         self._name = name
         self._scylla_version = scylla_version
-        self._io_properties = yaml.safe_load(self._get_io_properites_yaml())
 
-    @cache
-    def _get_io_properites_yaml(self):
-        return self.remoter.run('cat /etc/scylla.d/io_properties.yaml', retry=3).stdout
+    @cached_property
+    def _io_properties(self):
+        return yaml.safe_load(self.remoter.run('cat /etc/scylla.d/io_properties.yaml', verbose=False).stdout)
```

Comment:
Do you expect, that if command failed by any reason, the test will also failed? don't wont to use ignore_status=True?

Reply:
I catch it in some other place - this is the reason of this PR.

If run with ignore_status then it would cache wrong value and effectively deactivate feature.